LICENSE-MIT

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2017-2022 by the respective authors
```

Comment:
added this to try to be as inclusive as possible, maybe there's a better way but I don't know any.

Reply:
I think that works.